core/types/block.go

```diff
@@ -353,6 +353,9 @@ func CopyHeader(h *Header) *Header {
 	if h.Number != nil {
 		cpy.Number.Set(h.Number)
 	}
+	if h.BaseFee != nil {
+		cpy.BaseFee = new(big.Int).Set(h.BaseFee)
```

Comment:
what if h.BaseFee is nil?

Reply:
The cpy will still have a nil as baseFee. I don't know if I understand the question